Gemfile

```diff
@@ -13,17 +13,17 @@ else
 end
 
 gem 'rest-client', '>= 2.0.0', '< 3', :require => 'rest_client'
-gem 'audited', '~> 4.7'
+gem 'audited', '~> 4.7', '>= 4.7.1'
```

Comment:
The script I use to convert this to the spec file can't handle this well (it generates 3 Require statements). I'd prefer `'>= 4.7.1', '< 5'`.

Reply:
ok, changed :+1: